html/index.html

```diff
@@ -3,6 +3,15 @@
     <head>
         <meta charset="utf-8" />
         <title>Discord</title>
+        <script>
+            // Google Analytics
+            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+            ga('create', 'UA-49796218-29', 'auto');
```

Comment:
In a follow-up it might be nice to have this pull from an environment variable somehow.